aztec/src/test/kotlin/org/wordpress/aztec/AttributeTest.kt

```diff
@@ -0,0 +1,291 @@
+        private val NESTED_REVERSED = "<u class=\"klass\"><b><i a=\"A\">Nested</i></b></u>"
+        private val STRIKETHROUGH = "<s class=\"test\">Strikethrough</s>" // <s> or <strike> or <del>
+        private val ORDERED = "<ol l=\"L\"><li>Ordered</li></ol>"
+        private val UNORDERED = "<ul m=\"M\"><li>Unordered</li></ul>"
+        private val QUOTE = "<blockquote n=\"N\">Quote</blockquote>"
+        private val LINK = "<a o=\"O\" href=\"https://github.com/wordpress-mobile/WordPress-Aztec-Android\">Link</a>"
+        private val UNKNOWN = "<iframe class=\"classic\" p=\"P\">Menu</iframe>"
+        private val COMMENT = "<!--Comment--><br>"
+        private val COMMENT_MORE = "<!--more--><br>"
+        private val COMMENT_PAGE = "<!--nextpage--><br>"
+        private val LIST = "<ol><li a=\"1\">Ordered</li></ol>"
+        private val LIST_WITH_EMPTY_ITEMS = "<ul><li></li><li a=\"1\">1</li><li></li></ul>"
+        private val SUB = "<sub i=\"I\">Sub</sub>"
+        private val SUP = "<sup i=\"I\">Sup</sup>"
+        private val FONT = "<font i=\"I\">Font</font>"
+        private val TT = "<tt t=\"T\">Monospace</tt>"
+        private val BIG = "<big b=\"B\">Big</big>"
+        private val SMALL = "<small s=\"S\">Small</small>"
+        private val P = "<p p=\"P\">Paragraph</p>"
+        private val MIXED = HEADING + BOLD + ITALIC + UNDERLINE + STRIKETHROUGH + ORDERED +
+                UNORDERED + QUOTE + LINK + COMMENT + COMMENT_MORE + COMMENT_PAGE +
+                UNKNOWN + LIST + SUB + SUP + FONT + TT + BIG + SMALL + P
+    }
+
+    lateinit var editText: AztecText
+
+    /**
+     * Initialize variables.
+     */
+    @Before
+    fun init() {
+        val activity = Robolectric.buildActivity(Activity::class.java).create().visible().get()
+        editText = AztecText(activity)
+        activity.setContentView(editText)
+    }
+
+    @Test
+    @Throws(Exception::class)
+    fun headingAttributes() {
+        val input = HEADING
+        editText.fromHtml(input)
+        val output = editText.toHtml()
+        Assert.assertEquals(input, output)
+    }
+
+    @Test
+    @Throws(Exception::class)
+    fun italicAttributes() {
+        val input = ITALIC
+        editText.fromHtml(input)
+        val output = editText.toHtml()
+        Assert.assertEquals(input, output)
+    }
+
+    @Test
+    @Throws(Exception::class)
+    fun boldAttributes() {
+        val input = BOLD
+        editText.fromHtml(input)
+        val output = editText.toHtml()
+        Assert.assertEquals(input, output)
+    }
+
+    @Test
+    @Throws(Exception::class)
+    fun boldWithoutAttributes() {
+        val input = BOLD_NO_ATTRS
+        editText.fromHtml(input)
+        val output = editText.toHtml()
+        Assert.assertEquals(input, output)
+    }
+
+    @Test
+    @Throws(Exception::class)
+    fun nestedAttributes() {
+        val input = NESTED
+        val expected = NESTED_REVERSED
```

Comment:
Sounds good.  I just wanted to make sure we were tracking it.  I saw the new issue, https://github.com/wordpress-mobile/WordPress-Aztec-Android/issues/118.